menus/sync-settings.cson

```diff
@@ -8,6 +8,7 @@
         { 'label': 'Backup', 'command': 'sync-settings:backup' }
         { 'label': 'Restore', 'command': 'sync-settings:restore' }
         { 'label': 'View backup', 'command': 'sync-settings:view-backup' }
+        { 'label': 'Check for updated backup', 'command': 'sync-settings:check-backup' }
```

Comment:
Future enhancement it is - #104

For the name I think the current one, although long is most accurate.